src/sdkV2.ts

```diff
@@ -2300,6 +2272,114 @@ export default class CegaEvmSDKV2 {
     });
   }
 
+  async sfnSettleVault(
```

Comment:
i thought we discussed before not to include these functions, as we'll use the bulk function when performing these actions instead

Reply:
i guess it's fine to leave in if you want to

but it is unlikely we will use the single one if we have the option to use the bulk one